poseidon/build.rs

```diff
@@ -12,6 +12,7 @@ use poseidon_paramgen::poseidon_build;
 fn main() {
     let security_level = match env::var("SECURITY_LEVEL") {
         Ok(level) => match level.as_str() {
+            "32" => 32,
```

Comment:
This is not mentioned in the PR's description. Is it left here intentionally?

Reply:
Leftover, but harmless and potentially helpful